Libs/Visualization/VTK/Widgets/ctkVTKScalarsToColorsWidget.h

```diff
@@ -46,17 +46,17 @@ class CTK_VISUALIZATION_VTK_WIDGETS_EXPORT ctkVTKScalarsToColorsWidget : public
   ctkVTKScalarsToColorsWidget(QWidget* parent = 0);
   virtual ~ctkVTKScalarsToColorsWidget();
 
-  ctkVTKScalarsToColorsView* view()const;
-  vtkControlPointsItem* currentControlPointsItem()const;
+  Q_INVOKABLE ctkVTKScalarsToColorsView* view()const;
+  Q_INVOKABLE vtkControlPointsItem* currentControlPointsItem()const;
 
-  bool isHorizontalSliderVisible()const;
-  void setHorizontalSliderVisible(bool visible);
+  Q_INVOKABLE bool isHorizontalSliderVisible()const;
+  Q_INVOKABLE void setHorizontalSliderVisible(bool visible);
 
-  bool isVerticalSliderVisible()const;
-  void setVerticalSliderVisible(bool visible);
+  Q_INVOKABLE bool isVerticalSliderVisible()const;
+  Q_INVOKABLE void setVerticalSliderVisible(bool visible);
   
-  bool editColors()const;
-  void setEditColors(bool edit);
+  Q_INVOKABLE bool editColors()const;
+  Q_INVOKABLE void setEditColors(bool edit);
```

Comment:
Since this one is a boolean property, it could be added as:
```
Q_PROPERTY(bool editColors READ editColors WRITE setEditColors)
```

Reply:
In fact it is already wrapped using `Q_PROPERTY`, I gonna integrate removing the extra `Q_INVOKABLE`

Reply:
Same comment for `isHorizontalSliderVisible/setHorizontalSliderVisible` and `isVerticalSliderVisible/setVerticalSliderVisible`